extensions/systemd/index.js

```diff
@@ -41,7 +49,9 @@ class SystemdExtension extends cli.Extension {
             ghost_exec_path: process.argv.slice(0,2).join(' ')
         }), 'systemd service', serviceFilename, '/lib/systemd/system').then(
             () => this.ui.sudo('systemctl daemon-reload')
-        );
+        ).catch((error) => {
+            return Promise.reject(new cli.errors.ProcessError(error));
```

Comment:
In my opinion both `ctx.instance.template` and `this.ui.sudo` should return a custom error already.
So nobody has to handle errors if calling both units.

Reply:
Well, in some cases you might want to catch the error manually from sudo and do something with it. For instance, we catch any error from the sudo letsencrypt generation command via acme so that we can check the stderr. That would still be possible if `ui.sudo` caught and re-threw the error as a ProcessError, but it means additional logic.

At any rate, probably a subject for a later PR 😄